common/config.go

```diff
@@ -186,16 +199,19 @@ func DefaultMoneroPortFromEnv(env Environment) uint {
 	}
 }
 
-// ChainIDFromEnv returns the expected chainID that we should find on the
+// ChainNameFromEnv returns the expected chainID that we should find on the
 // ethereum endpoint when running int the passed environment.
-func ChainIDFromEnv(env Environment) *big.Int {
+func ChainNameFromEnv(env Environment) string {
 	switch env {
 	case Development:
-		return big.NewInt(GanacheChainID)
+		return "Ganache"
```

Comment:
i kind of prefer lower case as this is going in the protocol id with lowercase atomic-swap

Reply:
I made them all lower case.